src/main/python/systemds/script_building/dag.py

```diff
@@ -43,6 +43,8 @@ class OutputType(Enum):
     NONE = auto()
     SCALAR = auto()
     STRING = auto()
+    INT = auto()
+    BOOLEAN = auto()
```

Comment:
These are not valid, since both are instances of SCALAR,
where are they used?

Reply:
But isnt STRING also a value type of Scalar?

I added boolean and int there since in the scalar class, the value type of the scalar is determined by this. But you're right it would make more sense to put the value type in a different enum and create a new attribute, so that there is one field for the data type (matrix, frame, scalar) and one for the value types (double, integer, string, boolean)

Reply:
you are right, that would make more sense.

I think the reason why it is as it is now, is because someone (probably me) was lazy when making the initial version.

Reply:
I will continue on this PR once the other Datatype / Valuetype PR is gone through